Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +37,60 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            var sb = new StringBuilder();
+
+            sb.Append(AddBorder(message, padding));
+            sb.Append(AddPaddingLines(message, padding));
+            sb.Append(AddMessageLines(message, padding));
+            sb.Append(AddPaddingLines(message, padding));
+            sb.Append(AddBorder(message, padding));
+
+            return sb.ToString();
+        }
+
```

Comment:
nit: consider reusing similar parts of code:
whatever is similar should go under a function;
whatever is different should come as an argument.